tests/cpp/plugin/test_federated_communicator.cc

```diff
@@ -40,6 +40,9 @@ class FederatedCommunicatorTest : public ::testing::Test {
   }
 
   void TearDown() override {
+    while (!server_) {
```

Comment:
This seems odd, could you please share when it helps?

Reply:
This was somehow needed to get the create test to work. Seems more trouble than its worth. Removed.